tests/State/ScopeTest.php

```diff
@@ -273,7 +319,8 @@ public function testApplyToEvent(): void
         $scope->addBreadcrumb($breadcrumb);
         $scope->setTag('foo', 'bar');
         $scope->setExtra('bar', 'foo');
-        $scope->setUser(['foo' => 'baz']);
+        $merge = true;
```

Comment:
Instead of using a variabile just pass `true` as value of the 2nd argument of the `setUser` function